src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
```

Comment:
For readability, it might be better to have a new line to separate main code and import lines.